sdk/servicebus/Azure.Messaging.ServiceBus/tests/RuleManager/RuleManagerLiveTests.cs

```diff
@@ -0,0 +1,589 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Messaging.ServiceBus.Administration;
+using NUnit.Framework;
+
+namespace Azure.Messaging.ServiceBus.Tests.RuleManager
+{
+    public class RuleManagerLiveTests : ServiceBusLiveTestBase
+    {
+        [Test]
+        public async Task AddGetAndRemoveRules()
```

Comment:
That's a pretty large amount of functionality for a single test to be responsible for.   Should we consider breaking this out into three tests to isolate the Get, Add, and Remove scopes?

Reply:
Well each of them will still need to do the Add step, so I think it is probably okay to combine them.